conda_smithy/lint_recipe.py

```diff
@@ -104,15 +104,18 @@ def lintify(meta, recipe_dir=None, conda_forge=False):
     # 6: Selectors should be in a tidy form.
     if recipe_dir is not None and os.path.exists(meta_fname):
         bad_selectors = []
+        bad_lines = []
         # Good selectors look like ".*\s\s#\s[...]"
         good_selectors_pat = re.compile(r'(.+?)\s{2,}#\s\[(.+)\](?(2).*)$')
         with io.open(meta_fname, 'rt') as fh:
-            for selector_line in selector_lines(fh):
+            for selector_line, line_number in selector_lines(fh):
                 if not good_selectors_pat.match(selector_line):
                     bad_selectors.append(selector_line)
+                    bad_lines.append(line_number)
         if bad_selectors:
             lints.append('Selectors are suggested to take a '
-                         '``<two spaces>#<one space>[<expression>]`` form.')
+                         '``<two spaces>#<one space>[<expression>]`` form.'
+                         'see lines {}'.format(bad_lines))
```

Comment:
Minor thing I noticed (of course after seeing in production 😄) We should add a space `form.`

xref: https://github.com/conda-forge/mosh-feedstock/pull/5#issuecomment-345473080

Reply:
(A capital S would be good too!)

Reply:
Also it seems to be suffering an off by one error?